CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * Extracted `SlackRubyBot::Server` from `SlackRubyBot::App` - [@dblock](https://github.com/dblock).
 * Fix: explicitly require 'giphy' - [@dblock](https://github.com/dblock).
 * Fix: undefined method `stop` for `Slack::RealTime::Client` - [@dblock](https://github.com/dblock).
+* Fix: Bot failing to correctly respond to unknown commands when queried with format `@botname` - [@crayment](https://github.com/crayment)
```

Comment:
Nitpick: Update with a link to this PR (like below), please, and put a period at the end, it's a full sentence :)